### DISCUSSION TITLE
What if you added an individual script tag for every mod and the just removed it when removing the mod?

### DISCUSSION THREAD

**Discussion by stpv221** [eaglerforge/EaglerForge#23] [Ideas]
Would it work? imma try working on it myself

**Comment by leahanderson1** [comment]
it would be better for the mod to have a script that execs before it stops to unregister event listeners and shit

**Comment by stpv221** [comment]
alr

**Comment by ZXMushroom63** [comment]
removing the script tag wouldn't do anything, because the script has already run and added lsiteners.

**Comment by stpv221** [comment]
I got that
